aggregator/elem.go

```diff
@@ -392,9 +458,13 @@ func (e *TimerElem) Close() {
 	e.values = e.values[:0]
 	e.toConsume = e.toConsume[:0]
 	pool := e.opts.TimerElemPool()
+	aggTypesPool := e.opts.AggregationTypesPool()
+	quantileFloatsPool := e.opts.QuantileFloatsPool()
 	e.Unlock()
 
 	pool.Put(e)
+	aggTypesPool.Put(e.aggTypes)
```

Comment:
Same here, the internal fields need to be returned to pool before the element is returned.